data/schema/schema_v1.json

```diff
@@ -12,6 +12,18 @@
             "description": "A question code used by downstream systems to identify answers",
             "pattern": "^[0-9a-z]+$"
         },
+        "show_guidance": {
+            "description": "Allows customisation of guidance text",
+            "type": "string"
+        },
+        "hide_guidance": {
+            "description": "Allows customisation of guidance text",
+            "type": "string"
+        },
+        "content": {
+            "description": "Allows customisation of guidance text",
+            "type": "string"
+        },
```

Comment:
The schema needs some work to make the guidance content a common element, I've created a gist to demonstrate what this looks like - https://gist.github.com/ajmaddaford/6130dd89bd5939a02b3e7cb1053481d7. We can have a look together tomorrow and discuss.